crates/livesplit-hotkey/src/wasm_web/mod.rs

```diff
@@ -31,11 +34,37 @@ impl Drop for Hook {
                 "keypress",
                 self.keyboard_callback.as_ref().unchecked_ref(),
             );
-            window.clear_interval_with_handle(self.interval_id);
+            if let Some(interval_id) = self.interval_id.get() {
+                window.clear_interval_with_handle(interval_id);
```

Comment:
It's definitely an edge case, but should we also be calling this in `unregister` if there's no more gamepad buttons registered anymore?

Reply:
I don't think that case matters much, as I think it's a super rare situation that someone was using gamepad, then stops using gamepad. And even then, the cost for the gamepad hook is still very minimal, and it stops being a cost the next time they reload. So overall, I'd say yes, we could possibly remove the hook when unregistering all gamepad buttons, but it has a much lesser impact.